core/modules/simpletest/simpletest.module

```diff
@@ -368,18 +368,24 @@ function simpletest_test_get_all() {
       $check_keys = array('name', 'description', 'group', 'file');
       foreach ($files as $file) {
         $classes = backdrop_parse_info_file($file->uri, TRUE);
+        // Error messages will get translated as we create them below, but we
```

Comment:
I don't think we need all this documentation here about how `watchdog()` works - this was probably very useful for development, but should be removed before commit.

Reply:
OK